style/style.css

```
@@ -1,323 +1,785 @@
+  align-items: center;
+  display: flex;
+  height: 4.375rem;
+  border-bottom: 0.0625rem solid #DFDFDF;
+  padding: 0.5625rem 12.5rem;
+  gap: 0.625rem;
+  justify-content: space-between;
+  width: 100%;
+  z-index: 120;
+  background-color: #FFFFFF;
+}
+
+.logo {
+  width: 9.375rem;
+  height: auto;
+}
+
+.login-button {
+  width: 8rem;
+  padding: 0.625rem 1.25rem;
+  background-color: #3692FF;
+  color: white;
+  border: none;
+  border-radius: 0.3125rem;
+  font-size: 1rem;
+  cursor: pointer;
+  white-space: nowrap;
+  
+}
+
+.container0 {
+  display: flex;
+  width: 100%;
+  height: 33.75rem;
+  top: 4.375rem;
+  justify-content: center;
+  background-color: #CFE5FF;
+  gap:0.625rem;
+  align-items: end;
   
-  .logo {
-    width: 150px;
-    height: auto;
-    margin-left:63px;
-  }
   
-  .login-button {
-    padding: 10px 20px;
-    background-color: #3692FF;
-    color: white;
-    border: none;
-    border-radius: 5px;
-    font-size: 16px;
-    cursor: pointer;
-    margin-left: 20px;
-  }
-    
-.container0{ 
-    position: relative;
-    display:flex;
-    height:540px;
-    top:70px;
-    justify-content: space-around;
-    align-items: end;
-    background-color: #CFE5FF;
-    
 }
 
-  .inner_box0{
-  width:1110px;
-  height: 340px;
+.inner_box0 {
+  height: 21.25rem;
   display: flex;
-  align-items: center;
-  top:200px;
-  left:405px;
-  gap:7px;
+  top: 12.5rem;
+  left: 25.3125rem;
+  gap: 0.4375rem;
   
-  } 
   
-   .inner_box0 a{
-    text-decoration: none;
-   }
   
- .text-box0{
-    display: flex;
-    width:357px;
-    height:260px;
-    padding-bottom: 60px;
-    gap:32px;
- }
- .text-box0 h{
-    width: 295px;
-    height: 112px;
-    font-weight:700;
-    font-size:40px;
-    line-height: 56px;
-    letter-spacing:2%;
+}
 
+.inner_box0 a {
+  text-decoration: none;
 }
- 
-.see{
-    width:357;
-    height:56px;
-    border:1px solid;
-    border-radius: 40px;
-    background-color:#3692FF ;
-    color:#F9FAFB;
-    padding:16px 124px;
-    margin:46px 0;
-    gap:10px;
-    white-space: nowrap;
+
+.text-box0 {
+  /* width: 22.3125rem; */
+  height: 16.25rem;
+  padding-bottom: 3.75rem;
+  gap: 2rem;
 }
+
+.text-box0 h {
+  width: 18.4375rem;
+  height: 7rem;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.5rem;
+  letter-spacing: 2%;
+}
+
+.see {
+  width: 22.3125rem;
+  height: 3.5rem;
+  border: 0.0625rem solid;
+  border-radius: 2.5rem;
+  background-color: #3692FF;
+  color: #F9FAFB;
+  padding: 1rem 7.75rem;
+  margin: 2.875rem 0;
+  gap: 0.625rem;
+  white-space: nowrap;
+}
+
 .see p {
-  width:109px;
-  height: 32px;
+  width: 6.8125rem;
+  height: 2rem;
   font-family: Pretendard;
   font-weight: 600;
-  font-size: 20px;
-  line-height: 32px;
+  font-size: 1.25rem;
+  line-height: 2rem;
   letter-spacing: 0%;
   text-align: center;
   vertical-align: middle;
-
 }
 
-.contents0 img{
-  width: 746px;
-  height: 340px;
+.contents0 img {
+  width: 46.625rem;
+  height: 21.25rem;
```

❗️ 수정요청
이렇게 height, width 속성을 동시에 주게 되시면 이미지의 원본 비율을 고려해 작성해주신 것이 아니라면 지금처럼 비율이 달라지게 됩니다!
구현할때 이미지의 비율을 유지하는 것이 중요하므로 비율을 유지할 수 있는 방향으로 수정해보세요!

```suggestion
.contents0 img {
  width: 46.625rem;
  height: auto;
```